rllib/policy/torch_policy.py

```diff
@@ -124,15 +141,63 @@ def compute_log_likelihoods(self,
             if prev_reward_batch:
                 input_dict[SampleBatch.PREV_REWARDS] = prev_reward_batch
 
-            parameters, _ = self.model(input_dict, state_batches, [1])
+            parameters, _ = self.model(input_dict, state_batches, seq_lens)
             action_dist = self.dist_class(parameters, self.model)
             log_likelihoods = action_dist.logp(input_dict[SampleBatch.ACTIONS])
             return log_likelihoods
 
     @override(Policy)
     def learn_on_batch(self, postprocessed_batch):
-        train_batch = self._lazy_tensor_dict(postprocessed_batch)
+        shuffle = False
+        if shuffle:
```

Comment:
Isn't this always set to False in the line above?

Reply:
Maybe also comment this is copied from TF policy and vice versa (can we share the code in a util func)?

Reply:
Yeah, this `shuffle=False` is not properly done here. I'll move it into a util and try to share with tf.

Reply:
Done.